_pages/en/building-enabling-directorate.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Building Nested Enabling Teams
+ref: mandate
+lang: en
+status: posted
+categories: Work In Progress
+permalink: /building-enabling-directorate.html
+---
+
+## {{ page.title }}
+
+### Problem Statement
+
+In [Team Topologies](https://itrevolution.com/book/team-topologies/) they write that organizations should organize themselves by nesting the four team definitions.
+
+- Value Stream Aligned Teams
+- Platform Teams
+- Enabling Teams
+- Complex Subsystem Teams
+
+In our particular case, to use it as an example, the IT Strategy team is an enabling team. We are under a director who considers the directorate to be an enabling team.  Therefore the IT Strategy team is an enabling team within an enabling team, and has other sibling teams that are also enabling teams. That being the case, how would one synchronize activities of the entire directorate? Say the director was to offer their services to a team elsewhere in the organization. How best to manage and coordinate the work between the series of sub-enabling teams? That is the question this document is intending to answer.
```

Comment:
Clarify that the scope of this document is focused on one directorate: ITSM (or IT Service & Vendor Management).
Unless the scope is that of the IT Strategy team towards the rest of IITB?
Or is the scope that of the ITSM team towards the rest of IITB?

Reply:
My understanding is that IT Strategy is an enabling team, same as each other teams in ITSM. So that makes our directorate an enabling directorate (a team of enabling teams).

Reply:
I tried to make it general so other teams could leverage it as well. Though @gcharest was right ..

Unless the scope is that of the IT Strategy team towards the rest of IITB? Yes
Or is the scope that of the ITSM team towards the rest of IITB? Yes

SARA is an 'Enabling Team'
ITPSM is an 'Enabling Team'
The teams within ITPSM are 'Enabling Teams'

That's the power of the model in the book is that the definitions get recursively applied. 

But the focus on the article is how ITPSM will interact. And in that model I also attempt to describe how the teams will interact with each other, and other teams, as part of that model.